velox/vector/tests/VectorTest.cpp

```diff
@@ -3535,5 +3536,71 @@ TEST_F(VectorTest, hashAll) {
   }
 }
 
+TEST_F(VectorTest, setType) {
```

Comment:
Let's combine these 2 tests. The testing logic is exactly same, hence, no need to duplicate.

Reply:
@mbasmanova Combined.